cygnus-common/src/main/java/com/telefonica/iot/cygnus/utils/DataUtils.java

```diff
@@ -0,0 +1,89 @@
+/**
+ * 
+ */
+package com.telefonica.iot.cygnus.utils;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * @author PMO Santander Smart City - Ayuntamiento de Santander
+ *
+ */
+public final class DataUtils {
+
+    private static final String[] PATTERNS = {
```

Comment:
A brief comment above this line should explain the pattern. It is not evident just looking it :)

Reply:
I commented the pattern where I use different regular expressions